Platform/AlderlakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -29,9 +29,11 @@
 # sign_bin_flag can be set to false to avoid signing process. Applicable for Btg profile 0
 sign_bin_flag = True
 
-sblopen_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../../../../', 'SblOpen')
+sblopen_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../'))
```

Comment:
Since it is common for all platforms, it might be better to create a function in IfwiUtility.py so that everybody can just call it.  It is easier to maintain in this way.

Reply:
This is somewhat a chicken-and-egg problem :), since we could import IfwiUtility module only after the SBL root is identified, where either the user explicitly specifies it with SBL_SOURCE or it is assumed four levels up relative to the path of StitchIfwi.py

Reply:
OK, your point is valid.   I think we can do this for now.  
Maybe later on we can consider a generic StitchIfwi.py for all platforms, and this script can use modules for platform specific customization.  Just similar to how we deal with BuildLoader.py for all platforms.